gokart/task.py

```diff
@@ -279,6 +281,34 @@ def get_task_params(self) -> Dict:
             return self.load(target)
         return dict()
 
+    @luigi.Task.event_handler(luigi.Event.START)
+    def _set_random_seed(self):
+        random_seed = self._get_random_seed()
+        seeds = {
+            'seed': random_seed,
+            'seed_method': [
+                'random.seed',
+                'np.random.seed',
+            ]
+        }
+        random.seed(random_seed)
+        np.random.seed(random_seed)
+        try:
+            import torch
+            torch.manual_seed(random_seed)
+            seeds['seed_method'].append('torch.manual_seed')
+            torch.cuda.manual_seed(random_seed)
+            seeds['seed_method'].append('torch.cuda.manual_seed')
+        except Exception:
```

Comment:
I think ImportError should be used. How do you think?

Reply:
How do I add other modules in my local project?  Do I need to add codes to gokart?

Reply:
Good comment.  
I found two issues in response to your point.

1. add other modules in local project (nishiba-san say)
2. misleading if we want to fix the seed from outside

Change to WIP once.

Reply:
I add `fix_random_seed_methods` :)